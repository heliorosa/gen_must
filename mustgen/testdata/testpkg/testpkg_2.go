package testpkg

func DoStuff[T any]() (T, error) {
	//@gen_must
	var zero T
	return zero, nil
}
