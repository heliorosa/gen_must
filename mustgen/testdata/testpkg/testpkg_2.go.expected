// Code generated - DO NOT EDIT.
// This file is auto generated by gen_must and any manual changes will be lost.

package testpkg

// MustDoStuff has the behavior of DoStuff, except it panics on error
func MustDoStuff[T any]() T {
	var0, err := DoStuff[T]()
	if err != nil {
		panic(err)
	}
	return var0
}
